# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/webmin/nodefuser.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./setup.sh.orig	2005-01-24 19:15:16.000000000 -0300
+++ ./setup.sh	2005-02-12 11:19:41.308502117 -0300
@@ -438,7 +438,6 @@
 			echo "Webmin does not support being started at boot time on your system."
 		fi
 	fi
-	makeboot=$atboot
 
 	# Copy files to target directory
 	echo "***********************************************************************"
@@ -455,8 +454,8 @@
 	echo "Creating web server config files.."
 	cfile=$config_dir/miniserv.conf
 	echo "port=$port" >> $cfile
-	echo "root=$wadir" >> $cfile
-	echo "mimetypes=$wadir/mime.types" >> $cfile
+	echo "root=$prefix" >> $cfile
+	echo "mimetypes=$prefix/mime.types" >> $cfile
 	echo "addtype_cgi=internal/cgi" >> $cfile
 	echo "realm=Webmin Server" >> $cfile
 	echo "logfile=$var_dir/miniserv.log" >> $cfile
@@ -490,15 +489,7 @@
 	md5pass=`$perl -e 'print crypt("test", "\\$1\\$A9wB3O18\\$zaZgqrEmb9VNltWTL454R/") eq "\\$1\\$A9wB3O18\\$zaZgqrEmb9VNltWTL454R/" ? "1\n" : "0\n"'`
 
 	ufile=$config_dir/miniserv.users
-	if [ "$crypt" != "" ]; then
-		echo "$login:$crypt:0" > $ufile
-	else
-		if [ "$md5pass" = "1" ]; then
-			$perl -e 'print "$ARGV[0]:",crypt($ARGV[1], "\$1\$XXXXXXXX"),":0\n"' "$login" "$password" > $ufile
-		else
-			$perl -e 'print "$ARGV[0]:",crypt($ARGV[1], "XX"),":0\n"' "$login" "$password" > $ufile
-		fi
-	fi
+	touch $ufile
 	chmod 600 $ufile
 	echo "userfile=$ufile" >> $cfile
 
@@ -535,11 +526,7 @@
 	echo "Creating access control file.."
 	afile=$config_dir/webmin.acl
 	rm -f $afile
-	if [ "$defaultmods" = "" ]; then
-		echo "$login: $allmods" >> $afile
-	else
-		echo "$login: $defaultmods" >> $afile
-	fi
+	touch $afile
 	chmod 600 $afile
 	echo "..done"
 	echo ""
@@ -560,7 +547,7 @@
 echo "Creating start and stop scripts.."
 rm -f $config_dir/stop $config_dir/start $config_dir/restart
 echo "#!/bin/sh" >>$config_dir/start
-echo "echo Starting Webmin server in $wadir" >>$config_dir/start
+echo "echo Starting Webmin server in $prefix" >>$config_dir/start
 echo "trap '' 1" >>$config_dir/start
 echo "LANG=" >>$config_dir/start
 echo "export LANG" >>$config_dir/start
@@ -571,13 +558,13 @@
 echo "export PERLLIB" >>$config_dir/start
 uname -a | grep -i 'HP/*UX' >/dev/null
 if [ $? = "0" ]; then
-	echo "exec '$wadir/miniserv.pl' $config_dir/miniserv.conf &" >>$config_dir/start
+	echo "exec '$prefix/miniserv.pl' $config_dir/miniserv.conf &" >>$config_dir/start
 else
-	echo "exec '$wadir/miniserv.pl' $config_dir/miniserv.conf" >>$config_dir/start
+	echo "exec '$prefix/miniserv.pl' $config_dir/miniserv.conf" >>$config_dir/start
 fi
 
 echo "#!/bin/sh" >>$config_dir/stop
-echo "echo Stopping Webmin server in $wadir" >>$config_dir/stop
+echo "echo Stopping Webmin server in $prefix" >>$config_dir/stop
 echo "pidfile=\`grep \"^pidfile=\" $config_dir/miniserv.conf | sed -e 's/pidfile=//g'\`" >>$config_dir/stop
 echo "kill \`cat \$pidfile\`" >>$config_dir/stop
 
